ml/backend/ggml/ggml.go

```diff
@@ -476,6 +482,96 @@ func (b *Backend) Load(ctx context.Context, progress func(float32)) error {
 			}
 			defer file.Close()
 			sr := io.NewSectionReader(file, int64(b.meta.Tensors().Offset+t.Offset), int64(t.Size()))
+
+			// TODO this should use some other marker to toggle behavior so we can support loading type=4 and type=39
+			if t.Kind == 39 {
```

Comment:
I think we'll need to tighten this up before merging to avoid double transforming kind:39 types loaded from a GGUF that already had the transform applied by a conversion script.